package-lock.json

```diff
@@ -8,7 +8,6 @@
       "version": "2.1.3",
       "resolved": "https://registry.npmjs.org/@nodelib/fs.scandir/-/fs.scandir-2.1.3.tgz",
       "integrity": "sha512-eGmwYQn3gxo4r7jdQnkrrN6bY478C3P+a/y72IJukF8LjB6ZHeB3c+Ehacj3sYeSmUXGlnA67/PmbM9CVwL7Dw==",
-      "dev": true,
```

Comment:
ooi, how did you install the `npm` dependencies? I am kind of intrigued that this lock file keeps changing on us. There must be different ways on how people install npm dependencies.

Reply:
Just did npm install as per the Readme, weird.

Reply:
not an issue really,  but I have seen this file changing forth and back now. Something must be different somewhere.